MoreLinq/ToDelimitedString.cs

```diff
@@ -25,37 +25,14 @@ namespace MoreLinq
 
     static partial class MoreEnumerable
     {
-        /// <summary>
-        /// Creates a delimited string from a sequence of values. The
-        /// delimiter used depends on the current culture of the executing thread.
-        /// </summary>
-        /// <typeparam name="TSource">Type of element in the source sequence</typeparam>
-        /// <param name="source">The sequence of items to delimit. Each is converted to a string using the
-        /// simple ToString() conversion.</param>
-        /// <returns>
-        /// A string that consists of the elements in <paramref name="source"/>
-        /// delimited by <see cref="TextInfo.ListSeparator"/>. If the source
-        /// sequence is empty, the method returns an empty string.
-        /// </returns>
-        /// <remarks>
-        /// This operator uses immediate execution and effectively buffers the sequence.
-        /// </remarks>
-
-        [Obsolete]
-        public static string ToDelimitedString<TSource>(this IEnumerable<TSource> source)
-        {
-            return ToDelimitedString(source, null);
-        }
-
         /// <summary>
         /// Creates a delimited string from a sequence of values and
         /// a given delimiter.
         /// </summary>
         /// <typeparam name="TSource">Type of element in the source sequence</typeparam>
         /// <param name="source">The sequence of items to delimit. Each is converted to a string using the
         /// simple ToString() conversion.</param>
-        /// <param name="delimiter">The delimiter to inject between elements. May be null, in which case
-        /// the executing thread's current culture's list separator is used.</param>
+        /// <param name="delimiter">The delimiter to inject between elements.</param>
```

Comment:
The new `<exception>` tag for the delimiter should be added.

Reply:
If you insist :) although we haven't been doing that consistently.